subsys/net/ip/connection.c

```diff
@@ -554,6 +554,7 @@ enum net_verdict net_conn_input(struct net_pkt *pkt,
 	bool is_mcast_pkt = false, mcast_pkt_delivered = false;
```

Comment:
Also the commit subject `net: Add support for SOCK_RAW and AF_PACKET` is misleading as we already have support for AF_PACKET sockets. The subject should say something like `net: Add support for simultaneous UDP and raw sockets`, also commit body could be clarified a bit.

Reply:
Ok.